quickfixj-messages/quickfixj-messages-fix44/pom.xml

```diff
@@ -27,7 +27,7 @@
 							<goal>generate</goal>
 						</goals>
 						<configuration>
-							<dictFile>./src/main/resources/FIX44.modified.xml</dictFile>
```

Comment:
The xml file? Probably not, I didn't wish to delete it as it's the standard FIX definition that comes with QuickFix. Downside to keeping it is you have to delete the other FIX files to build the binaries correctly. 

If you're referring to the `pom.xml` changes, I'd prefer to keep them so it's easier to build but the change is simple enough to make at build time so not necessary to keep.